src/test/scala/com/databricks/spark/avro/AvroWriteBenchmark.scala

```diff
@@ -49,7 +50,8 @@ object AvroWriteBenchmark {
 
   private def generateRandomRow(): Row = {
     val rand = new Random()
-    Row(rand.nextString(defaultSize), rand.nextInt(), new Date(rand.nextLong()) ,rand.nextDouble(), rand.nextDouble(),
+    Row(rand.nextString(defaultSize), rand.nextInt(), new Date(rand.nextLong()) ,rand.nextDouble(),
```

Comment:
Can you also update this
`new Date(rand.nextLong()) ,rand.nextDouble(),`
to
`new Date(rand.nextLong()), rand.nextDouble(),`

After that , I will merge this PR.